Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/riskcards/RiskCardNoInternetStateProvider.kt

```diff
@@ -0,0 +1,37 @@
+package de.rki.coronawarnapp.ui.main.riskcards
+
+import dagger.Reusable
+import de.rki.coronawarnapp.storage.RiskLevelRepository
+import de.rki.coronawarnapp.storage.TracingRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.onCompletion
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.flow.onStart
+import timber.log.Timber
+import javax.inject.Inject
+
+@Reusable
+class RiskCardNoInternetStateProvider @Inject constructor(
+    tracingRepository: TracingRepository
+) {
+
+    val state: Flow<RiskCardNoInternet> = combine(
+        RiskLevelRepository.riskLevelScoreLastSuccessfulCalculated.onEach {
+            Timber.v("riskLevelScoreLastSuccessfulCalculated: $it")
```

Comment:
do you think we can use natural words here instead ofusing camel case variables?
More of a question than a suggestion.

Reply:
It might be a good idea. I just followed the other state providers as an example for this (SubmissionCardsStateProvider) and they all have the Timber messages like this. Right now it looks like the Timber logs differ from person to person and  I don't really think we have some guidelines for them.